### PR TITLE
[jsk_tools] Add Documentation for rossetip,rossetlocal,rossetmaster

### PR DESCRIPTION
**Pull request by wkentaro**

### CODE REVIEW

File: jsk_tools/README.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+```sh
+# rossetmaster ${hostname} ${ros_port}
+# default: hostname=pr1040, ros_port=11311
+user@host $ rossetmaster
+set ROS_MASTER_URI to http://pr1040:11311
+[http://pr1040:11311] user@host $ echo $ROS_MASTER_URI
+http://pr1040:11311
+```
+
+The default robot hostname is `pr1040`, you can override
```
Comment (garaemon) [inline jsk_tools/README.md]: I don't want to recommend this manner

Comment (garaemon) [inline jsk_tools/README.md]: I'd like to recomment to create new rossetfoo alias like rossetpr1040

Comment (wkentaro) [inline jsk_tools/README.md]: I see. That's simpler.

**Comment by wkentaro** [comment]
I updated.

**Comment by garaemon** [comment]
:+1:

**Pull request merged**